packages/spatial/src/physics/components/RigidBodyComponent.tsx

```diff
@@ -139,11 +139,15 @@ const RigidBodyReactor = () => {
   const component = useComponent(entity, RigidBodyComponent)
   const physicsWorld = Physics.useWorld(entity)!
 
+  useEffect(() => {
```

Comment:
while i would like to get to the bottom of this (i suspect it could be oddities in the physics functions & physics systems) this seems fine for now